archetype-java-builders/src/main/resources/archetype-resources/src/main/java/ExampleJLoadScenarioProvider.java

```diff
@@ -103,8 +113,9 @@ public JLoadScenario exampleJaggerLoadScenario() {
 
         // For JLoadScenario which is supposed to be executed by Jagger its ID must be set to 'jagger.load.scenario.id.to.execute' property's value
         return JLoadScenario.builder(Id.of("exampleJaggerLoadScenario"), jParallelTestsGroup)
-                            .addListener(new ExampleLoadScenarioListener())
```

Comment:
It wasn't me. Git has decided that I removed it. I just added percentiles. :ok_man: